modules/nf-core/atlasgeneannotationmanipulation/gtf2featureannotation/main.nf

```diff
@@ -0,0 +1,43 @@
+process ATLASGENEANNOTATIONMANIPULATION_GTF2FEATUREANNOTATION {
+    tag '$gtf'
```

Comment:
```suggestion
    tag '${meta.id}'
```
This would be good in print the case/sample id in the log files in case someone wants to run this process multiple times in the same pipeline, no?